Example/Podfile

```diff
@@ -4,3 +4,9 @@ use_frameworks!
 target 'ReusableDemo' do
   pod 'Reusable', :path => '../'
 end
+
+post_install do |installer|
+  installer.pods_project.build_configurations.each do |config|
```

Comment:
Reminder: you'll probably be able to get rid of that when you'll migrate to CP 1.1.1